### PR TITLE
Added topic for company identities on testnet

### LINKED ISSUE #428: Document how to obtain test "company identities"
**Task description**

We currently document how to get a company identity on mainnet, but every exchange we are integrating first needs this for testnet as well.

Until now I've sent them instructions individually each time, but it would be good to have a place to link directly.

The process is essentially the same, except
- they need to use testnet configuration files.
[enterprise-identities-dry-run.zip](https://github.com/Concordium/concordium.github.io/files/8196573/enterprise-identities-dry-run.zip)
- they need to send an email to somebody at concordium. Me, Caro or Emil B have handled these requests in the past, but there are instructions (https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/identity-provider-cli.md ) that anybody that can use command-line tools can follow.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

We need to describe the process of how to obtain a company identity on testnet as more exchanges start to integrate.

## Changes

Added topic for company identities to testnet and adjusted content.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

Closes #428

### CODE REVIEW

File: source/testnet/net/guides/company-identities.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - SHA256 checksum of the download: ``6ea2674ebae5dafd9de3c730db536fc0675627b6b867f05a944a1a60dd5ceca8``
+
+   - `Windows <https://distribution.concordium.software/tools/windows/signed/concordium-client_3.0.4-0.exe>`_
+      - SHA256 checksum of the download: ``f6cafdd472b02ead818cc0f463a4cf40053dda33a824a2dfed816744a48a579c``
+
+   - `MacOS <https://distribution.concordium.software/tools/macos/signed/concordium-client_3.0.4-0.zip>`_
+      - SHA256 checksum of the download: ``3173cbe28373d9a787978b236aefdaa20d129496b61e545ed7369d8922e10d05``
+
+
+#. To generate a request for an identity object, follow the `generate request instructions <https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#generate-a-request-for-the-identity-objectinstructions>`_. Email the ``request.json`` output file to support@concordium.software. Store the auxiliary output securely.
```
Comment (abizjak) [inline source/testnet/net/guides/company-identities.rst:48]: Should we tell them to add a specific subject to speed up the request?

File: source/testnet/net/guides/company-identities.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#. When the identity has been verified successfully, Concordium will notify you by email, and they will send you an identity object file named ``id-object.json``.
```
Comment (abizjak) [inline source/testnet/net/guides/company-identities.rst:50]: We will also create the initial account.

**Review by abizjak — APPROVED** [review]

File: source/testnet/net/guides/company-identities.rst (hunk 2 above)
Comment (dg-concordium) [reply source/testnet/net/guides/company-identities.rst]: Added text for this.

File: source/testnet/net/guides/company-identities.rst (hunk 1 above)
Comment (dg-concordium) [reply source/testnet/net/guides/company-identities.rst]: Yes, added a subject line text for them.

**Pull request merged**